package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@newfold-labs/wp-module-migration",
```

Comment:
We should define engines: https://github.com/newfold-labs/wp-plugin-hostgator/blob/main/package.json#L14
We should also add an `.nvmrc` file that sets the Node version: https://github.com/newfold-labs/wp-plugin-hostgator/blob/main/.nvmrc